include/nsysccr/cdc.h

```diff
@@ -23,11 +24,23 @@ typedef struct CCRCDCFWInfo CCRCDCFWInfo;
 typedef struct CCRCDCSoftwareVersion CCRCDCSoftwareVersion;
 typedef struct CCRCDCDrhState CCRCDCDrhState;
 typedef struct CCRCDCDrcState CCRCDCDrcState;
+typedef struct CCRCDCRegisterCallbackData CCRCDCRegisterCallbackData;
 typedef uint8_t CCRCDCDestination;
 typedef uint32_t CCRCDCWpsStatusType;
 typedef uint8_t CCRCDCWakeState;
 typedef uint8_t CCRCDCUicConfigId;
 
+struct WUT_PACKED CCRCDCRegisterCallbackData {
+        int32_t attached;
+        VPADChan chan;
```

Comment:
Hmm, not sure if we should pull in the vpad/vpadbase include for this, since vpadbase is using nsysccr internally.
We could just keep this as a `uint32_t` or add a channel type for nsysccr.

Reply:
changed into a uint32_t